report-ng/app/src/value-converters/chip-name-value-converter.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Testerra
+ *
+ * (C) 2023, Selina Natschke, Deutsche Telekom MMS GmbH, Deutsche Telekom AG
+ *
+ * Deutsche Telekom AG and all other contributors /
+ * copyright owners license this file to you under the Apache
+ * License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import {autoinject} from "aurelia-framework";
+import {StatusConverter} from "../services/status-converter";
+import {ClassName, ClassNameValueConverter} from "./class-name-value-converter";
+import {FilterType, IFilterChip} from "../components/classes/classes";
+
+@autoinject()
+export class ChipNameValueConverter {
+
+    constructor(
+        private _statusConverter: StatusConverter,
+        private _classNameValueConverter: ClassNameValueConverter
+    ) {
+    }
+
+    toView(filterChip: IFilterChip, converterType: ChipConverterType) {
+
+        if (converterType == ChipConverterType.TOOLTIP) {
+
+            switch (filterChip.filter.type) {
```

Comment:
Because all tool tips are static texts, they can moved to array `_chips: IFilterChip`.
This simplifies the converter and `ChipConverterType` can be removed.